### PR TITLE
main screen top block changes

### PR DESCRIPTION
**Pull request by Romanist**

### CODE REVIEW

File: src/scripts/index.js [hunk 1]
```diff
@@ -973,3 +973,35 @@ $(document).ready(function () {
...
+    // Time calculations for days, hours, minutes and seconds
+    var days = Math.floor(distance / (1000 * 60 * 60 * 24))
+    var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
+    var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60))
+    // var seconds = Math.floor((distance % (1000 * 60)) / 1000)
+
+    // Display the result in the element with id="demo"
+    // document.getElementById("demo").innerHTML = days + "d " + hours + "h " + minutes + "m " + seconds + "s "
+    $('.top_timer__d .timerJs').html(days)
+    $('.top_timer__h .timerJs').html(hours)
```
Comment (rumkin) [inline src/scripts/index.js]: Hours and minutes are not padded with zeros.

File: src/scripts/index.js [hunk 2]
```diff
@@ -973,3 +973,35 @@ $(document).ready(function () {
     }
   })
 })
+
+// timer
+$(document).ready(function () {
+  var countDownDate = new Date('Dec 10, 2018 00:00:00').getTime()
+  var x = setInterval(function () {
```
Comment (rumkin) [inline src/scripts/index.js]: Name `x` as `interval`.

File: src/scripts/index.js [hunk 3]
```diff
@@ -973,3 +973,35 @@ $(document).ready(function () {
...
+    // Get todays date and time
+    var now = new Date().getTime()
+
+    // Find the distance between now and the count down date
+    var distance = countDownDate - now
+
+    // Time calculations for days, hours, minutes and seconds
+    var days = Math.floor(distance / (1000 * 60 * 60 * 24))
```
Comment (rumkin) [inline src/scripts/index.js]: Define precalculated variables `MINUTE`, `HOUR` and `DAY` and use it like this:

```js
var days = Math.floor(distance / DAY);
var hours = Math.floor((distance % DAY) / HOUR)
```

File: src/views/components/top.pug [hunk 4]
```diff
@@ -1,14 +1,57 @@
 .top-wrapper.js-section
+  //- script(type="application/javascript", async="async", src="https://icoholder.com/en/widget/big-green/21042.js?width=300")
```
Comment (rumkin) [inline src/views/components/top.pug]: Why commented code is in PR? Should it be removed or uncommented?

File: src/views/components/top.pug [hunk 5]
```diff
@@ -1,14 +1,57 @@
...
+          span.top_timer-colored :
+        .top_timer__h
+          span.top-bold_700.timerJs 22
+          | h
+          span.top_timer-colored :
+        .top_timer__m
+          span.top-bold_700.timerJs 59
+          | m
+    .top__widgets
+      // a(href="https://icomarks.com/ico/membrana", target="_blank", rel="nofollow", title="Membrana ICO").top__widget1.top__widget
```
Comment (rumkin) [inline src/views/components/top.pug]: Another unknown comment.

File: src/views/components/top.pug [hunk 6]
```diff
@@ -1,14 +1,57 @@
...
+      //   //- img(border="0", src="https://icomarks.com/widget/m/membrana/horizontal.svg", width="320px", height="100px", alt="Membrana ICO Rating")
+      //   img(border="0", src="/static/img/icons/mark.svg", width="320", height="100", alt="Membrana ICO Rating").top__widget1_img
+
+      a(href="https://icobench.com/ico/membrana" target="_blank" rel="nofollow" title="Membrana on ICObench").top__widget1.top__widget
+        img(border="0" src="https://icobench.com/rated/membrana?shape=horizontal&size=s" width="174px" height="76px" alt="Membrana ICO rating")
+
+      a(href="https://www.trackico.io/ico/membrana/", target="_blank", title="Membrana on TrackICO").top__widget1.top__widget
+        img(border="0", src="https://www.trackico.io/widget/horizontal/membrana/300.png", width="230px", height="76px", alt="Membrana TrackICO rating")
+
```
Comment (rumkin) [inline src/views/components/top.pug]: Another unknown comment.

File: src/views/components/top.pug (hunk 4 above)
Comment (Romanist) [reply src/views/components/top.pug]: it is from task with widgets here. This widget wasnt working cause of icoholder.com crashing and i uncluded it but commented to use it after when u probably will ask to do it again

File: src/views/components/top.pug (hunk 5 above)
Comment (Romanist) [reply src/views/components/top.pug]: this widget was layout-like but after u've asked to change it for another one. I'm not sure it wont be used again so i just saved it like this. If no feel free to remove it

File: src/views/components/top.pug (hunk 6 above)
Comment (Romanist) [reply src/views/components/top.pug]: ?

File: src/scripts/index.js (hunk 2 above)
Comment (Romanist) [reply src/scripts/index.js]: it was example. My bad. Didnt notice

File: src/scripts/index.js (hunk 3 above)
Comment (Romanist) [reply src/scripts/index.js]: omw

File: src/views/components/top.pug (hunk 6 above)
Comment (rumkin) [reply src/views/components/top.pug]: At line 39

**Pull request merged**